dotnet/UnityBridge/UnityConverters.cs

```diff
@@ -1,11 +1,15 @@
 ﻿using System;
+using System.IO;
 using System.Collections.Generic;
 using Ara3D;
+using Ara3D.Revit.DataModel;
```

Comment:
So we don't need this anymore, this coupling shouldn't be here.

Reply:
Correct. This dependency will be removed when the UnityLib branch is merged into `vimaec/ara3d-dev > develop`. This pull request will be on hold until that happens.

Reply:
The Ara3D.Revit.DataModel dependency has been removed.